web/src/enterprise/campaigns/detail/patches/PatchNode.tsx

```diff
@@ -0,0 +1,138 @@
+import classNames from 'classnames'
+import { FileDiffConnection } from '../../../../components/diff/FileDiffConnection'
+import { FilteredConnectionQueryArgs } from '../../../../components/FilteredConnection'
+
+export interface PatchNodeProps extends ThemeProps {
+    node: IPatch
+    campaignUpdates?: Subject<void>
+    history: H.History
+    location: H.Location
+    /** Shows the publish button */
+    enablePublishing: boolean
+}
+
+export const PatchNode: React.FunctionComponent<PatchNodeProps> = ({
+    node,
+    campaignUpdates,
+    isLightTheme,
+    history,
+    location,
+    enablePublishing,
+}) => {
+    const [isPublishing, setIsPublishing] = useState<boolean>()
+    useEffect(() => {
+        setIsPublishing(node.publicationEnqueued)
+    }, [node.publicationEnqueued])
+    const [publishError, setPublishError] = useState<Error>()
+    const publishChangeset: React.MouseEventHandler = async () => {
+        try {
+            setPublishError(undefined)
+            setIsPublishing(true)
+            await _publishChangeset(node.id)
+            if (campaignUpdates) {
+                campaignUpdates.next()
+            }
+        } catch (error) {
+            setPublishError(asError(error))
+        } finally {
+            setIsPublishing(false)
+        }
+    }
+    const fileDiffs = node.diff?.fileDiffs
+
+    const changesetNodeRow = (
+        <div className="d-flex align-items-start m-1 ml-2">
+            <div className="changeset-node__content flex-fill">
+                <div className="d-flex flex-column">
+                    <div>
+                        <Octicon icon={Diff} className="icon-inline mr-2" />
+                        <strong>
+                            <Link
+                                to={node.repository.url}
+                                className={classNames(node.__typename === 'Patch' && 'text-muted')}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                {node.repository.name}
+                            </Link>
+                        </strong>
+                    </div>
+                </div>
+            </div>
+            <div className="flex-shrink-0 flex-grow-0 ml-1 align-items-end">
+                {fileDiffs && <DiffStat {...fileDiffs.diffStat} expandedCounts={true} />}
+            </div>
+            {enablePublishing && (
+                <>
+                    {publishError && <ErrorIcon data-tooltip={publishError.message} className="ml-2" />}
+                    <button
+                        type="button"
+                        className="flex-shrink-0 flex-grow-0 btn btn-sm btn-secondary ml-2"
+                        disabled={isPublishing}
+                        onClick={publishChangeset}
+                    >
+                        {isPublishing && <LoadingSpinner className="mr-1 icon-inline" />}{' '}
```

Comment:
Either the `mr` or the space has no effect, I'm guessing the `mr`
```suggestion
                        {isPublishing && <LoadingSpinner className="icon-inline" />}{' '}
```